prog/dftb+/prg_dftb/dftbplus.F90

```diff
@@ -451,23 +451,30 @@ program dftbplus
 
     if (tSocket) then
       call socket%receive(coord0, latvec)
```

Comment:
What about introducing a kind of buffer array receiving the lattice vectors from the socket? We could transfer its content into `latVec` if the system is periodic. This would make the `else` branch and all changes in `initprogram.F90` superfluous, and additionally prevent that `latVec` contains garbage.